doc/nrf/glossary.rst

```diff
@@ -0,0 +1,447 @@
+
+Real-time operating system (RTOS)
+   An operating system that reacts to input within a specific period of time.
+   A real-time deadline can be so small that system reaction appears instantaneous.
+
+Real-Time Transfer (RTT)
+   Proprietary technology for bidirectional communication that supports J-Link devices and Arm-based microcontrollers, developed by SEGGER Microcontroller.
+
+Receive Data (RXD)
+   A signal line in a serial interface that receives data from another device.
+
+Repository
+   The entire set of files and folders of which a project is composed, together with the revision history of each file.
+   It is often composed of multiple branches.
+   It is also known as *Git repository* or *Git project*.
+
+Root of Trust (RoT)
+   The property or component in a secure system that provides the foundation of a Chain of Trust.
+
+Sample
+   An implementation example that showcases a single feature or library.
+
+Secure Access Port Protection (SECUREAPPROTECT)
+   A register used to prevent read and write access to all secure CPU registers and secure memory-mapped addresses.
+   See Access Port Protection (APPROTECT).
+
+Security Manager Protocol (SMP)
+   A protocol used for pairing and key distribution.
+
+SEGGER Embedded Studio (SES)
+   A cross-platform Integrated Development Environment (IDE) for embedded C/C++ programming with support for Nordic Semiconductor devices, produced by SEGGER Microcontroller.
+   The |NCS| uses a custom :ref:`Nordic Edition of SES <installing_ses>`.
+
+Signal-to-Noise Ratio (SNR)
+   The level of signal power compared to the level of noise power, often expressed in decibels (dB).
+
+Simple Network Time Protocol (SNTP)
+   A less complex implementation of NTP, using the same protocol but without requiring the storage of state over extended periods of time.
+
+Soft fork
+   A fork that contains a very small set of changes when compared to its upstream.
+
+SoftDevice
+   A wireless protocol stack that complements the nRF5 Series SoCs.
+   Nordic Semiconductor provides these stacks as qualified, precompiled binary files.
+
+SoftDevice Manager (SDM)
+   A SoftDevice component that controls the SoftDevice state and configures the behavior of certain core functionality.
+
+Software Development Kit (SDK)
+   A set of tools used for developing applications for a specific device or operating system.
+
+Spinel
+   A general management protocol for enabling a host device to communicate with and manage co-processors, like a network co-processor (NCP) or a radio co-processor (RCP).
+
+System in Package (SiP)
+   Several integrated circuits, often from different technologies, enclosed in a single module that performs as a system or subsystem.
+
+System on Chip (SoC)
+   A microchip that integrates all the necessary electronic circuits and components of a computer or other electronic systems on a single integrated circuit.
+
+System Protection Unit (SPU)
```

Comment:
Isn't the SPU a nordic-only peripheral? The explanation seems right, but maybe we should mention that. As it stands now it looks like it is a general concept.

Reply:
fixed